src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(Shape.CIRCLE, color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.round(Math.PI * this.radius * this.radius);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(this);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
You should not override toString in this task. Fix it also in other figures

![image](https://user-images.githubusercontent.com/117547096/211056779-a51231ba-0eb4-42cd-8dbb-680fc6a8ed98.png)